common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -567,6 +567,14 @@ def should_force_account_creation():
                 (current_provider.skip_email_verification or current_provider.send_to_registration_first))
 
     if not user:
+        details = kwargs.get('details', {})
```

Comment:
Instead of doing this, specify a `details` kwarg in the function signature above.